### ISSUE TITLE
Aerials/Maps Test Pre-Ingest

### ISSUE THREAD

**Issue by rmjaffe** [UCSCLibrary/dams_project_mgmt#416]
## Collection Information

This is a test ingest of a single flight from our aerial photos collection and a map selected from the UCSC Library Maps set.  The purpose of this test load is for us to get a sense of what these materials look like in the current DAMS environment so as to be able make an informed decision about whether to abandon Cdm or not.

### Objects

* Type of content (images, audio, albums, etc.): 2 complex image objects

### Metadata Preparation

* [x] Object-level metadata created / remediated
* [x] Object-level metadata formatted per ingest spreadsheet specification
* [x] Spreadsheet available on DOMTP
* [x] DI notified

* Spreadsheet filename: Maps_Aerials_Test_Ingest_20201207.xlsx
* Special requirements / issues:

### File Assessment / Technical Processing 

* [x] Files assessed - match spreadsheet filenames
* [x] Digitization quality assessed
* [x] Files moved to dams_ingest

* Directory name: /dams_ingest/maps_test
* Special requirements / issues:

### Pre-Ingest

* [x] Metadata spreadsheet(s) moved to dams_ingest
* [x] Ned notified; tickets created for collection ingest, QA work, and publication

**Comment by rmjaffe** [comment]
@chesleyperry If you want to select the two test objects, please do.  Just let me know which and I can put together the test ingest spreadsheet.

**Comment by chesleyperry** [comment]
My nomination for the test aerial flight:  1943.  I was looking for a flight that was not too large (this one has 156 images) with only one index.  Most of the individual sheets have geonames locations.  The representative image for the series should be the index: 1943_cja_index_geo.jpg.  @rmjaffe - if that works for you, I will begin copying files to dams_ingest.

**Comment by rmjaffe** [comment]
@chesleyperry That sounds like a good candidate to me.  I'll copy that flight onto a separate spreadsheet. Do we also want to do one of the Standard atlases or 'official' maps?

**Comment by chesleyperry** [comment]
If possible, I'd like to do 2 map tests.  First is the Atlas of Santa Cruz County, California, 1929 edition from the Standard Map Service Atlases collection.  It has 60 images, mp0001_map_0060.tif - mp0001_map_0107.tif.

The other is a single image - the "Official map of Santa Cruz County: full map", filename is Hatch_Full_Map.tif.  In CONTENTdm, this one is available as a single image and then broken into 30 tiles.  If the test works with the full map and allows sufficient Zooming capabilities, we won't have to load the tiles.

**Comment by chesleyperry** [comment]
@rmjaffe 
> Do we also want to do one of the Standard atlases or 'official' maps? Perhaps this guy: http://digitalcollections.ucsc.edu/digital/collection/p15130coll3/id/1698/rec/4?

For that one - I'd like to have Angelika try to take the 6 tiles and combine them into one image.  She hasn't started on yet though so it isn't likely a good test subject.

**Comment by rmjaffe** [comment]
@chesleyperry @NedHenry  Just to confirm -- these test objects will be removed after they've been tested, correct?  The metadata for the Standard Atlas and the Hatch Map is messy, incomplete, and likely not wholly accurate either.  The aerial photo flight is also missing some collection level data despite the flight and sheet level metadata having been remediated.

**Comment by chesleyperry** [comment]
@rmjaffe  -- That's ok with me.  Since this is a test - we may want to define the success criteria specifically for the test. For example - I'm using this test to evaluate the UV to make sure it displays the appropriate level of detail for the large maps.  I also want to explore the structure for the aerial photos and make sure the parent and child works (and flight indexes) display in a logical way for patrons. I'm assuming the results of the test will inform any specialized actions we need to take when we ingest the larger collections.
Is it appropriate to include those assessment criteria in this ticket?

**Comment by rmjaffe** [comment]
@chesleyperry That works for me.  As far as the metadata, I'm mostly curious about the structure.  The aerial flight looks like this: aerials collection (collection)-->flight (work)-->sheet (work); whereas the UCSC Library Maps are a bit different.  The Standard Atlas example: library maps collection (collection)-->standard atlas collection (collection)-->atlas (work)-->page (work).  This is equivalent to a Fackler scrapbook.

But the Hatch Map is different from anything else we've ingested; the simple object work is a member of the collection of collections: library maps collection (collection)-->map (work).  This will be true for most of those maps unless we want to create and join them to an artificial parent 'collection'.  Must works be part of a collection in order to be part of a collection of collections?  I'm curious to see if this map ingests successfully ingested and if so, how it will look.

I'm just about done with the metadata -- I'm going to supply some placeholder values now that I know objects and metadata will be removed after we've finished testing.
